Cargo.toml

```diff
@@ -129,9 +129,10 @@ goauth = { version = "0.9.0", optional = true }
 smpl_jwt = { version = "0.6.1", optional = true }
 
 # API
-async-graphql = { version = "=2.5.0", optional = true }
-async-graphql-warp = { version = "=2.5.0", optional = true }
+async-graphql = { version = "=2.4.11", optional = true }
```

Comment:
Downgraded (and stapled) to 2.4.11, to unblock a fix for subscription context. Will be able to bump to the latest when #6383 lands.